### PR TITLE
Update pickem links to new domain

### PR DESCRIPTION
**Pull request by hallowatcher**
Hello!

The pickem domain is changing from [https://pickem.hwchr.com](https://pickem.hwchr.com) to [https://pickem.hwc.hr](https://pickem.hwc.hr) so I thought I would update all references to point to the new URL.

I also added a "hosted by" next to the pickem link, as this was done for TWC so I thought I would do it for all references.

### CODE REVIEW

**Comment by cl8n** [comment]
since when do wiki pages exclude things on basis of not being "needed" 🤔 more info is fine and that's a good place to show it considering we're asking people to click an external link

**Comment by TPGPL** [comment]
I see it as the same thing as other links, so why arent we adding "hosted by Google" for spreadsheets, "hosted by Challonge" for brackets etc. then

**Comment by hallowatcher** [comment]
Cuz im an individual from the community hosting these for the tourneys

Like I said, it was added by you guys for CWC (already in master), so I added it everywhere else as well. Don't really see the issue but if we gotta remove it then it's okay as well

**Comment by cl8n** [comment]
- sometimes that is part of the link text already
- link titles can do the same thing and lately we are making better use of them
- pointing out where things are created by osu! community members feels nice
- not everything needs to be so formulaic .-.

**Comment by TPGPL** [comment]
im fine with leaving the credit in the articles although i dont think repeating the same information in all articles is a good solution (adding a seperate article about the page could be a good way of solving the problem maybe?)

**Comment by TPGPL** [comment]
oh right we can add it as link hover text instead thats a good solution

File: wiki/Contests/oBWC/2/en.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ The osu! Beatmapping World Championship is run by various community members.
 - [Discord server](https://discord.gg/CZp4bNx)
 - [Official website](https://obwc.net/)
 - [Twitter](https://twitter.com/osubwc)
-- [Pick'em page](https://pickem.hwchr.com/tournaments/30)
+- [Pick'ems page](https://pickem.hwc.hr/tournaments/30) hosted by ![][flag_DE] [hallowatcher](https://osu.ppy.sh/users/1874761)
```
Comment (TPGPL) [inline wiki/Contests/oBWC/2/en.md:62]:
```suggestion
- [Pick'ems page "hosted by osu! community member hallowatcher"](https://pickem.hwc.hr/tournaments/30)
```
or something could be fine?

**Comment by hallowatcher** [comment]
Would've prefered to leave the credit as a link
Tooltips aren't mobile friendly

Anyways, this isn't too big of a deal, maybe you guys can clarify between each other

Screenshot_20200702-010914_Chrome

**Comment by TPGPL** [comment]
im fine leaving it how it is now then, maybe my reaction was exaggerated because of how much space it was taking in the code view dunno ¯\_(ツ)_/¯

**Comment by hallowatcher** [comment]
any updates on this last pending review? should I just not edit the news post?

**Pull request merged**